tests/src/FunctionalJavascript/ListPagesPluginTest.php

```diff
@@ -64,8 +69,8 @@ public function testListPagePluginForm(): void {
     // Edit the node and assert that we show correct values in the form.
     $this->drupalGet($node->toUrl('edit-form'));
     $this->clickLink('List Page');
-    $this->assertOptionSelected('Source entity type', 'Taxonomy term');
-    $this->assertOptionSelected('Source bundle', 'Vocabulary one');
+    $this->assertSession()->optionExists('Source entity type', 'Taxonomy term');
+    $this->assertSession()->optionExists('Source bundle', 'Vocabulary one');
```

Comment:
Checking of 'selected' attribute is missed.

Reply:
This will be fixed on next update.